anchore_engine/analyzers/manager.py

```diff
@@ -0,0 +1,120 @@
+import os
```

Comment:
Is this just a wholesale move? or where there more changes?

Reply:
just a move, it's not strictly necessary for the fix since I decided to use the explicit config rather than just known package paths.